models/intel/action-recognition-0001/README.md

```diff
@@ -61,5 +61,11 @@ An embedding image, name: `0`, shape: `1, 16, 512` in the format `B, T, C`, wher
 
 The model outputs a tensor with the shape `1, 400`, each row is a logits vector of performed actions.
 
+## Demo usage
+
+The model can be used in the following demos provided by the OpenVINO toolkit to show its capabilities:
```

Comment:
```suggestion
The model can be used in the following demos provided by the Open Model Zoo to show its capabilities:
```

Reply:
suggest to change to Open Model Zoo who is actually provides the demos

Reply:
Updated